sql/core/src/main/scala/org/apache/spark/sql/execution/limit.scala

```diff
@@ -358,7 +358,8 @@ case class TakeOrderedAndProjectExec(
     val orderByString = truncatedString(sortOrder, "[", ",", "]", maxFields)
     val outputString = truncatedString(output, "[", ",", "]", maxFields)
 
-    s"TakeOrderedAndProject(limit=$limit, orderBy=$orderByString, output=$outputString)"
+    s"TakeOrderedAndProject(limit=$limit, offset=$offset, " +
```

Comment:
I think we should probably handle the case when `offset` is not set? It would show it as `0`.

Reply:
> I think we should probably handle the case when `offset` is not set? It would show it as `0`.

Thank you for your review. I will handle the case when `offset` is not set.

Reply:
> I think we should probably handle the case when `offset` is not set? It would show it as `0`.

@HyukjinKwon Thank you for your review. I handled this case. Please review again.